src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,25 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+public class Deadline extends Task {
+    public String dl;
+    public LocalDate exactTime;
```

Comment:
For the sake of information hiding (an OOP principle), fields of the class should be private. In the case when we need to access the field data, we can simply create a getter method that returns the data.